include/deal.II/lac/lapack_full_matrix.h

```diff
@@ -141,6 +141,18 @@ class LAPACKFullMatrix : public TransposeTable<number>
   void add (const number                    a,
             const LAPACKFullMatrix<number> &A);
 
+  /**
+   * Perform a rank-1 update of a symmetric matrix
+   * $ A \leftarrow A + a \, \rm v \rm v^T $.
+   *
+   * This function also works for Cholesky factorization. Updating ($a>0$) is
```

Comment:
Is the description of the up/downdating procedure only applicable if the matrix actually represents a Cholesky factorization?

If so, maybe start the second sentence with "In that case, updating ..."

Reply:
correct, will update.